src/server/routes/monthData.js

```diff
@@ -54,14 +99,35 @@ function selectClientMonthData(req, res) {
       console.error(`Status Code: ${error.statusCode}, Error Message: ${error.code}`);
 
       res.json({
-        statusCode: error.statusCode,
+        statusCode: error.statusCode || 500,
         errorMessage: error.code,
       });
     });
 }
 
+/**
+ * selectClientMonthDataPost(req, res, next)
+ * Handles the requests from the form submit button (when no JS).
+ * It redirects to the route to execute the function for server side requesting.
+ */
+function selectClientMonthDataPost(req, res) {
```

Comment:
I think you mean `Server` instead of `Client`?

Reply:
I use `Client` because the form is from client, but yeah the real call is made by server side.

Reply:
The form submission is a POST so it's a server side since it refreshes the page. When we do ajax calls, those are client side since we request a specific endpoint to just get json back. That name sounds better.